### PR TITLE
Update KB to utilize public TPM

### PR DESCRIPTION
**Pull request by aaronsr**
Updated KB to utilize the public KB URL for the ticket priority matrix instead of the zendesk link.

### CODE REVIEW

**Comment by srveit** [comment]
@aaronsr It appears that you are referencing a nonexistant page

**Comment by jim-phillips** [comment]
@aaronsr Actually, you are just missing the .md extension of `Support/ticket-prioritzation-matrix`

File: Support/what-can-you-expect-from-centurylink-cloud-on-an-incident.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 </ul>
 <h3>"Ticket updates"</h3>
 <ol>
-  <li>All issues are worked in accordance with our prioritization matrix that can be found here: &nbsp;<a href="https://t3n.zendesk.com/entries/21651149-Ticket-Prioritization-Matrix">https://t3n.zendesk.com/entries/21651149-Ticket-Prioritization-Matrix</a>&nbsp;</li>
+  <li>All issues are worked in accordance with our prioritization matrix that can be found here: &nbsp;<a href="./ticket-prioritization-matrix">Ticket Prioritization Matrix</a>&nbsp;</li>
```
Comment (srveit) [inline Support/what-can-you-expect-from-centurylink-cloud-on-an-incident.md]: Change href to `./ticket-prioritization-matrix.md`

**Comment by rpdubose** [comment]
@aaronsr If you can, pls correct by EOW or we'll need to close this one.

**Pull request closed without merging**